backend/src/main/java/ai/verta/modeldb/batchProcess/CollaboratorResourceMigration.java

```diff
@@ -83,7 +84,10 @@ private static void migrateProjects() {
         Root<ProjectEntity> root = criteriaQuery.from(ProjectEntity.class);
 
         CriteriaQuery<ProjectEntity> selectQuery =
-            criteriaQuery.select(root).orderBy(criteriaBuilder.asc(root.get("id")));
+            criteriaQuery
+                .select(root)
+                .where(criteriaBuilder.equal(root.get("visibility_migration"), false))
```

Comment:
Should we add an index on this?

Reply:
Added index for the visibility_migration column